### ISSUE TITLE
[1.11.2] JEI/Optifine Bug

### ISSUE THREAD

**Issue by MakroxMiau** [sokratis12GR/ArmorPlus#101]
**ArmorPlus Version**: armorplus-1.11.2-9.5.6.7-beta

**TheDragonLib Version**: thedragonlib-1.11.2-2.2.3

**MinecraftForge Version**: forge 1.11.2

**Minecraft Version**: 1.11.2

**Client or Server?**:
- [x] Client
- [ ] Server

**Crash Report (via. [Pastebin](http://pastebin.com/))**: N/A

**What happens / happened?**: JEI/Optifine bug 

**How can I reproduce it?**: Have in the mods folder the following mods: armorplus, thedragonlib, jei, optifine and open the inventory of jei and look for the armorplus page, when you find it will not load and crash

**(Optional) Mod List**:

**Comment by sokratis12GR** [comment]
Could you provide me the crashlog pasted in Pastebin or in paste.ee ?

**Comment by sokratis12GR** [comment]
actually, what version of JEI are you using ?

**Comment by MakroxMiau** [comment]
idk how to get the crashlog in the new launcher, im using the lastest version of jei

**Comment by sokratis12GR** [comment]
Do the following: Press `Windows Key` + `R`  or search and launch "run" then inside type: `%appdata%/.minecraft/` and inside `crashlogs/` the latest one should be the right one

**Comment by MakroxMiau** [comment]
Oh, that makes sense. (google translate xd)

I think it's this http://pastebin.com/ewmmM5gM

**Comment by sokratis12GR** [comment]
Hmm strange the crashlog doesn't seem to have anything related to armorplus...

**Comment by MakroxMiau** [comment]
what do you see in the crashlog?

**Comment by sokratis12GR** [comment]
I see rendering problems `java.lang.IllegalStateException: Already building!
    at net.minecraft.client.renderer.VertexBuffer.func_181668_a(VertexBuffer.java:291)
    at net.minecraft.client.shader.Framebuffer.func_178038_a(Framebuffer.java:264)
    at net.minecraft.client.shader.Framebuffer.func_147615_c(Framebuffer.java:229)
    at net.minecraft.client.Minecraft.redirect$renderFBO$zzf000(Minecraft.java:3238)
    at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:1104)
    at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:372)
    at net.minecraft.client.main.Main.main(SourceFile:124)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
    at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
    at net.minecraft.launchwrapper.Launch.main(Launch.java:28)`

**Comment by MakroxMiau** [comment]
Does that mean it's a pc problem?

**Comment by sokratis12GR** [comment]
No, but Its not an ArmorPlus related issue

**Comment by MakroxMiau** [comment]
Then why i put the armorplus pag the jei crash my minecraft? (Dat english)

**Comment by sokratis12GR** [comment]
Reason: #105
TL;DR updating optifine version should fix it
